packages/coinstac-ui/app/main/index.js

```diff
@@ -124,6 +136,65 @@ loadConfig()
       mainWindow.webContents.send(BAD_TOKEN);
     });
 
+    /**
+     * IPC Listener to prepare consortia files
+     */
+    ipcMain.on('prepare-consortia-files', async (event, { userId, fileTree, appDirectory }) => {
+      const userRunDirectory = path.join(appDirectory, 'runs', userId);
+
+      if (!fs.existsSync(userRunDirectory)) {
+        fs.mkdirSync(userRunDirectory, { recursive: true });
+      }
+
+      const res = [];
+
+      fileTree.forEach((consortium) => {
+        const consortiumDirectory = path.join(userRunDirectory, consortium.name);
+
+        if (!fs.existsSync(consortiumDirectory)) {
+          fs.mkdirSync(consortiumDirectory);
+        }
```

Comment:
async:-)

Reply:
any reason to use async?

Reply:
`fs.exists` is deprecated and `fsPromise` does not have `exists`.
we have to use `fs.existsSync` and if so, I would like to use `fs.mkdirSync` for consistency.

btw, I don't see any needs to use async here. @rssk